### ISSUE TITLE
Service message: should be a check for make important service message visible also in submit tickets and other pages.

### ISSUE THREAD

**Issue by PeopleInside** [mkoch227/Mods-for-HESK#372]
For important maintenance message ... can you implement a check box for not only show a message in the home page of the Help Desk but to all page?

So if a Service message is very important user can check this box to show in all Help Desk pages.

Should be great because user who reach the Help Desk from  direct department link can't see the Service Message.

**Comment by FormidableMulk** [comment]
I'd like this too.

As is, I re-use the snippet that queries for and displays the service message on the front-end homepage on the ticket submission form.  Our service messages are more relevant on the ticket submission form than anywhere else.  (It would be nice if they were configurable per-message though!)

**Comment by mkoch227** [comment]
I agree as well, however submit the idea to the uservoice page to see how others react to the suggestion as well.

**Comment by PeopleInside** [comment]
I think this is a very important improvement and not needs to be posted in user voice. As you can understand Service Message as work now can be not "safe" because is easy an user can skip the message.

Very easy if you have direct link to the support department or customer click on the emails for see the ticket online.. service message are currently showed only in the Main page..

But ok i have posted in uservoice page as you request. 
https://mods-for-hesk.uservoice.com/forums/254758-general/suggestions/10243644-service-message-improvments

**Comment by mkoch227** [comment]
> I think this is a very important improvement

Since you find a use for this feature, you see it as very important.  However those who don't use service messages at all would probably think that it's not that important :wink: Thanks for posting on the UserVoice :smile:
